File: packages/3/ms-3-expressjs/src/environments/stage.ts
import { IConfig } from '@gmahechas/erp-common-ms-utils-js';

export const config: IConfig = ({
	environment: process.env.ENVIRONMENT,
	'ms-3': {
		app: {
			name: process.env.APP_NAME,
			port: process.env.APP_PORT,
		},
		databases: {
			mongo: {
				uri: process.env.MS_THREE_DATABASES_MONGO_URI
			}
		}
	}
});

File: packages/3/ms-3-flaskpy/flaskr/environments/stage.py
import os

config = {
    "environment": os.environ["ENVIRONMENT"],
    "ms-3": {
        "app": {
            "name": "",
            "port": 5000,
        },
        "databases": {
            "mongo": {
                "uri": None
            }
        }
    }
}
